### ISSUE TITLE
Decompress (InflateZipFile) problem 

### ISSUE THREAD

**Issue by Sacristan** [StefH/UZipDotNet#2]
# world_20150824203931.zip

//built this file with -> http://hastebin.com/ilewapacur.avrasm

Exception: File directory signature error
UZipDotNet.InflateZipFile.ReadZipFileDirectory () (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:403)
UZipDotNet.InflateZipFile.OpenZipFile (System.String readFileName) (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:110)
# test.zip

//same files archived with 7zip

Exception: File directory signature error
UZipDotNet.InflateZipFile.ReadZipFileDirectory () (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:403)
UZipDotNet.InflateZipFile.OpenZipFile (System.String readFileName) (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:110)
# fish_config-master.zip

//downloaded repository from github as zip (same problem)

Exception: File directory signature error
UZipDotNet.InflateZipFile.ReadZipFileDirectory () (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:403)
UZipDotNet.InflateZipFile.OpenZipFile (System.String readFileName) (at Assets/Voxel Engine/Utils/UZipDotNet/InflateZipFile.cs:110)

Same issue persists when using demo code sample -> https://github.com/StefH/UZipDotNet/blob/master/UZipDotNet/src/UZipDotNet.ConsoleApp/Program.cs

When changing path - same issue.

Basically cant get past    using (var inf = new UZipDotNet.InflateZipFile(@filePath))

System:
OS Name Microsoft Windows 10 Pro
Version 10.0.10240 Build 10240

**Comment by Sacristan** [comment]
And here are the files themselves:
https://dl.dropboxusercontent.com/u/26408378/UZipDotNetIssue/fish_config-master.zip
https://dl.dropboxusercontent.com/u/26408378/UZipDotNetIssue/test.zip
https://dl.dropboxusercontent.com/u/26408378/UZipDotNetIssue/world_20150824203931.zip

Can You look at this ?

**Comment by StefH** [comment]
What .NET version do you use?

**Comment by Sacristan** [comment]
Actually im trying to port it to Unity.

So that will be .NET 2.0

**Comment by StefH** [comment]
I've added .NET 2.0, can you please test it in this project ?
Or use the latest from nuget ?
https://www.nuget.org/packages/UZipDotNet/0.0.4

**Comment by StefH** [comment]
Are you able to test this?

**Comment by Sacristan** [comment]
I chose a different library to achieve what i wanted. But - sure i can try testing this, but it can take a while when i have a spare moment to do so

**Comment by StefH** [comment]
Ok, fine. In case you have time, please test and open new issue if needed.
